fabtests/functional/atomic_verify.h

```diff
@@ -0,0 +1,4 @@
+
+extern int do_verification;
+int atomic_verification_setup(enum fi_datatype datatype, int jrank, void * buf, size_t buf_size);
+int atomic_verification_check(enum fi_datatype datatype, enum fi_op op, int jrank, void * buf, size_t buf_size);
```

Comment:
I probably should just put these couple of definitions inside rdm_atomic rather than it's own header file.